ext/objspace/objspace_dump.c

```diff
@@ -213,6 +215,10 @@ dump_object(VALUE obj, struct dump_config *dc)
     dc->cur_obj_references = 0;
     dc->cur_obj_klass = BUILTIN_TYPE(obj) == T_NODE ? 0 : RBASIC_CLASS(obj);
 
+    if (dc->partial_dump && (!ainfo || ainfo->generation < dc->since)) {
```

Comment:
I this conditional correct?  Shouldn't it be `>=`?

Reply:
The `if` body is an early return. So the condition in "inversed".

Reply:
Ah, got it 🤦🏻‍♂️